rust/routee-compass-powertrain/Cargo.toml

```diff
@@ -25,8 +22,16 @@ serde = { workspace = true, features = ["derive"] }
 serde_repr = "0.1"
 serde_json = { workspace = true }
 ndarray = "0.15"
-ort = { version = "1.15.3", optional = true }
+ort = { version = "2.0.0-alpha.2", optional = true }
 rayon = { workspace = true }
 
 [features]
-onnx = ["ort"]
+default = ["onnx"]
+onnx = ["ort", "ureq", "flate2", "tar", "zip", "sha256"]
```

Comment:
if the default is onnx, then how do we build this without onnx? is that like `features = []` or something to override the default?